source/educators/references/grading/gradebook_assignment_types.rst

```diff
@@ -1,11 +1,15 @@
 .. _Gradebook Assignment Types:
 
-******************************
+##########################
 Gradebook Assignment Types
-******************************
+##########################
 
 .. tags:: educator, reference
 
+.. list-table::
```

Comment:
What is the purpose of an empty table here?

Reply:
This is for adding ToC which is not included in the original doc.

Reply:
I'm fixing those errors by deleting the table.